test/unit_tests/16_prime_sieve_large/code_gen.py

```diff
@@ -0,0 +1,231 @@
+import sys
+import re
+import math
+import random
+
+def prime_gen():
+    lower = 2
+    upper = 10000
+
+    prime_cnt = 0
+    results = [1]
+    for num in range(lower, upper + 1):
+        # all prime numbers are greater than 1
+        if num > 1:
+            for i in range(2, num):
+                if (num % i) == 0:
+                    break
+            else:
+                results.append(num)
+                prime_cnt = prime_cnt + 1
+        if (prime_cnt == 500):
+            break
+
+    print("prime_cnt = %d" %(prime_cnt))
+    return results
+
+
+
+
+def main():
+    prime_nums = prime_gen()
+
+    rows = 9 + 1 # row 0 is reserved
+    cols = 40 + 1
+
+    f = open("aie.mlir", "w+")
+    # declare tile, column by row
+
+
+    f.write("""//===- aie.mlir ------------------------------------------------*- MLIR -*-===//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// (c) Copyright 2021 Xilinx Inc.
+//
+//===----------------------------------------------------------------------===//
```

Comment:
It's generally good practice to have a comment that automatically generated files are automatically generated.

Reply:
ok will do it!